src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -694,6 +694,8 @@ Parameter Plugin::GetMetric(const std::string& name, const std::map<std::string,
     std::string device_id = GetConfig(ov::device::id.name(), options);
 
     auto iter = device_map.find(device_id);
+    if (iter == device_map.end())
+        iter = device_map.find(std::to_string(cldnn::debug_configuration::device_id));
```

Comment:
This is for the situation where we cannot specify target device such as accuracy checker... The correct way would be to change accuracy checker but this will be easier path.

Reply:
Note from the meeting after discussion with Vladimir: @tuxedcat 
* Let's not expose DeviceID in debug_caps. You can just remove the parsing part I think
* Let's remove topology_test